src/Test/Utilities/Portable/CompilationVerifier.cs

```diff
@@ -180,7 +186,37 @@ public void VerifyTypeIL(string typeName, string expected)
                     Assert.True(found, "Could not find type named " + typeName);
                 }
             }
-            AssertEx.AssertEqualToleratingWhitespaceDifferences(expected, output.ToString(), escapeQuotes: false);
+
+            var outputString = output.ToString();
+            if (normalize)
+            {
+                expected = NormalizePlatformSpecificCode(expected);
```

Comment:
Let's keep the `expected` verbatim and just normalize the output.

Reply:
This will make things difficult for people developing on .Net Core, as they will need to put in the expected output for .net framework

Reply:
Scrap that, I'm talking nonsense